css-view-transitions-2/Overview.bs

```diff
@@ -263,72 +263,81 @@ document.startViewTransition({update: updateTheDOMSomehow});
 
 The ''@view-transition'' rule is used by a document to indicate that cross-document navigations
 should setup and activate a {{ViewTransition}}. To take effect, it must be present in the old document
-when unloading, and in the new document when it is [=ready to render=].
+when unloading, and in the new document when it is [=reveal=].
 
 
 ## @view-transition rule grammar ## {#view-transition-grammar}
 
 The ''@view-transition'' rule has the following syntax:
 
 <pre class=prod>
-	@view-transition same-origin {
+	@view-transition {
 		<<declaration-rule-list>>
 	}
 </pre>
 
-Note: currently <code>same-origin</code> has to be present, as a future placeholder for customizing
-the navigation behavior based on the URL.
-
 Note: as per default behavior, the ''@view-transition'' rule can be nested inside a
 [=conditional group rule=] such as ''@media'' or ''@supports''.
 
-## The [=@view-transition/trigger=] property ## {#view-transition-trigger-descriptor}
+## The [=@view-transition/navigation=] descriptor ## {#view-transition-navigation-descriptor}
 
 	<pre class='descdef'>
-	Name: trigger
+	Name: navigation
 	For: @view-transition
-	Value: navigation | none
+	Value: auto | none
 	Initial: none
 	</pre>
 
-	The '<dfn for="@view-transition">trigger</dfn>' property opts in to automatically starting a view transition when performing a navigation.
-	It needs to be enabled both in the old document (when unloading) and in the new document (when ready to render).
+	The '<dfn for="@view-transition">navigation</dfn>' descriptor opts in to automatically starting a view transition when performing a navigation of a certain type.
+	It needs to be enabled both in the old document (when unloading) and in the new document (when reveal).
 
-	<dl dfn-type=value dfn-for="trigger">
+	<dl dfn-type=value dfn-for="@view-transition/navigation">
 		: <dfn>none</dfn>
 		:: There will be no transition.
 
-		: <dfn>navigation</dfn>
+		: <dfn>auto</dfn>
 		:: The transition will be enabled if the navigation is same-origin, without cross-origin
-			redirects.
+			redirects, and is not a {{NavigationType/reload}}.
 	</dl>
 
+## The [=@view-transition/type=] descriptor ## {#view-transition-type-descriptor}
 
+	<pre class='descdef'>
+	Name: type
+	For: @view-transition
+	Value: <<custom-ident>>+
+	Initial: n/a
```

Comment:
In `@font-face`, there are  coupe of properties that specify N/A as initial value, but I think the reason they can do that is that the `@font-face` rule is invalid unless those properties are specified (`font-family` and `src`). The other properties can be omitted so they have an initial value (like `font-weight`). 

Here, `type` is an optional thing, so I think it needs to have an initial value. WDYT? Maybe we can make Value be something like `none | <<custom-ident>>+`. It makes it a bit awkward for JS since we should then make `none` an invalid type

But all of this is a very special `@-rule` stuff, not the regular cascade, so maybe this is ok too

Reply:
Discussed offline, I think this is fine.